src/rpcclient/tests/test_capture_fd.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import pytest
+
+from rpcclient.capture_fd import CaptureFD
+
+TEST_DATA: bytes = b"testtesttest"
```

Comment:
```suggestion
TEST_DATA: bytes = b'testtesttest'
```

As a convention, this project prefers `'` for strings